### DISCUSSION TITLE
Public page/form not working

### DISCUSSION THREAD

**Discussion by YTDcontrol** [Budibase/budibase#6296] [Help]
I'm working in a self hosted server in a docker container. I´m working on release 195. Followed instructions in documentation, but no matter i do always redirect to uth/login.
Aplication publushed.
Aplication no locked.
screens seted to public.

Answer #4280 nether works for me.

Regards.

**Comment by melohagan** [comment]
Hi @YTDcontrol 

Would you mind sending your app export to support@budibase.com please?
I can take a look then.

Just to verify, the URL you are using is the published one - i.e. it does not contain the word `dev`

**Comment by melohagan** [comment]
Thanks @YTDcontrol I've been able to replicate this bug and I've raised an issue here: https://github.com/Budibase/budibase/issues/6314
